src/main/resources/db/migration/V2022_05_25__YSMOD-328-saksbehandling.sql

```diff
@@ -0,0 +1,79 @@
+CREATE TABLE sak
+(
+    sak_id               BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    sak_status           VARCHAR(50)  NOT NULL,
+    opprettet_tidspunkt  TIMESTAMP    NOT NULL,
+    opprettet_av         VARCHAR(255) NOT NULL,
+    sakstype             VARCHAR(50)  NOT NULL,
+    aktoer_id            VARCHAR(20)  NOT NULL, -- brukes av flere systemer i NAV
+    bruker_identifikator VARCHAR(12)  NOT NULL, -- fnr/dnr
+    bruker_fornavn       VARCHAR(100) NOT NULL,
```

Comment:
bør alle brukerfeltene ligge rett på sak?

Reply:
Ja. Det er by design for å kunne iverata historiske data for navn og id